ui/splash/src/androidMain/AndroidManifest.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
Why is this permission in this module? Permissions should be bundled in the app itself for simplicity